### ISSUE TITLE
Is this dead?

### ISSUE THREAD

**Issue by ghost** [foxnewsnetwork/ember-redux-data#1]
Have you found something else?

**Comment by foxnewsnetwork** [comment]
Yes, GraphQL has effectively swallowed up the world of REST and consumed much of the problems originally encountered by ember-data and JSONAPI. Personally, I recommend you look into https://github.com/bgentry/ember-apollo-client as an entry way into the world of GraphQL for ember.

**Comment by ghost** [comment]
Ok. Thanks. What do you think about redux-orm: https://github.com/tommikaikkonen/redux-orm?

**Comment by foxnewsnetwork** [comment]
redux-orm is pretty solid, however, tommikaikkonen has stopped actively working on redux-orm and the project has instead been handed off to the community as per this 

https://github.com/tommikaikkonen/redux-orm/issues/123

it's possible that there are still active contributions to redux-orm, but it's far more likely you'll have to support redux-orm on your own if you chose to go with it.

My personally recommendation is that it's safe to go with since it's pretty much your only option for a client-side ORM (since the current meta is to just make graphQL request each time)
